src/Scene/NodeProcess.js

```diff
@@ -21,6 +21,7 @@ function NodeProcess(scene, camera, ellipsoid) {
 
     this.r = ellipsoid.size || new THREE.Vector3();
     this.cV = new THREE.Vector3();
+    this.cameraViewMatrix = new THREE.Matrix4();
```

Comment:
yo can move this.cameraViewMatrix in camera 
`camera.viewMatrix`
and add function `camera.updateViewMatrix()`